content/guides/bake/index.md

```diff
@@ -0,0 +1,499 @@
+---
+title: Mastering multi-platform builds, testing, and more with Docker Buildx Bake
+linkTitle: Mastering Docker Buildx Bake
+description: >
+  Learn how to manage simple and complex build configurations with Buildx Bake.
+summary: >
+  Learn to automate Docker builds and testing with declarative configurations using Buildx Bake.
+languages: [go]
+subjects: [devops]
+levels: [advanced]
+params:
+  time: 30 minutes
```

Comment:
Could this be not automated based on word count? I recall we add smth like this with Jekyll

Reply:
Yeah it's possible. We used to have it on all pages by default, but it wasn't very useful. And now the thinking had been that, since these are follow-along guides, maybe it isn't very representative. But I'm not sure the manual ones are very accurate either. And if we think the automatic estimations based on word count are too low, we could probably just 2x them for guides. I'll take a look

Reply:
Maybe make a custom one that would take multiple things into account:
* word count (1x factor)
* links (2x factor)
* code blocks (3x factor)

Reply:
some pages have videos, which makes this extra tricky :-/ We can't easily factor in the video length into the reading time, only option is probably to hard-code the video length in somewhere. Feels a bit too hacky.